frontend/src/components/Posts/FilterContainer.tsx

```diff
@@ -31,6 +34,7 @@ type FilterContainerProps<TMultiple = boolean> = TMultiple extends true
 
 export const FilterContainer = ({
   name,
+  header,
```

Comment:
What's the purpose of `header` prop?

Is it because we need to know which page we are on?